### DISCUSSION TITLE
Community AMA: Education is in session

### DISCUSSION THREAD

**Discussion by tuves** [community/community#17798] [GitHub Education]
![AMA Banner](https://user-images.githubusercontent.com/49915749/164445312-3dc07004-89bb-462e-9d66-ffe21e179c0f.png)

Where and how do I learn to code? Where can I find technical communities to get involved with? Where can I find technical communities to get involved with? Join me and [@juanpflores](https://github.community/u/juanpflores)[](https://github.community/t/community-ama-education-is-in-session/246312#no-registration-necessary-1) for April’s Community AMA: Education is in session! Bring all your education questions, we’ll provide the answers. 🤓 

No registration necessary!
Simply join us on [Twitch ](http://twitch.tv/GitHub)on April 28 at 11 AM PST

If you can’t make the event, drop your question(s) in this discussion and we’ll be sure to answer them.

**Comment by suprstarrd** [comment]
* How did GitHub Education come to be? 
* What impact do you think GitHub Education has had on students, teachers, and campuses?
* What impact has GitHub Education had on you?
* What got you into coding?
* What's the number one thing you can recommend for people jumping into code for the first time?

**Reply by tuves** [reply]
Thank you for the questions, @jbmagination!

**Comment by ghost** [comment]
Como que programadores iniciantes que moram no interior podem começar no mercado de trabalho?

**Comment by Micheal-jennifer** [comment]
Please who can help me with a command and token

**Comment by ghost** [comment]
I am new to Github, any pointers how to use it get in contact
